README.md

```diff
@@ -402,11 +402,11 @@ storage:
 ```
 See [examples/docker-compose-sqlite-storage](.examples/docker-compose-sqlite-storage) for an example.
 
-- If `storage.type` is `postgres`, `storage.path` must be the connection URL:
+- If `storage.type` is `postgres` or `mysql`, `storage.path` must be the connection URL:
 ```yaml
 storage:
-  type: postgres
-  path: "postgres://user:password@127.0.0.1:5432/gatus?sslmode=disable"
+  type: postgres # or mysql
+  path: "postgres://user:password@127.0.0.1:5432/gatus?sslmode=disable" # or user:password@tcp(127.0.0.1:3306)/gatus?charset=utf8mb4&parseTime=True&loc=Local&multiStatements=true&interpolateParams=true
```

Comment:
Could you make a config separate from this specifically for mysql? Instead of mixing the two in the same config